pyanaconda/dbus/dbus_constants.py

```diff
@@ -0,0 +1,44 @@
+#
+# dbus/constants.py: Anaconda DBUS constants
```

Comment:
My worry is that it's easy to confuse with the "normal" constant I presume we will continue to use. It's quite likely you will import both and then will have to make sure to prevent a collision (import as or importing individual constants).

Having a different name might not be elegant but prevents this collision.